elastic/kibana
# [ResponseOps][Alerting] Rules and Connectors search does not filter exact name

Resolves https://github.com/elastic/kibana/issues/138147

## Summary

I updated the Rules list search to search on exact matches using `literal` instead of `wildcard`.


### Checklist



### To verify

- Create rules with spaces in the name 
- Search on the name, and verify that the results returned make sense
- Try doing some other searching and verify that search is working as expected.
